### ISSUE TITLE
missing evaluate.py for closed-loop simulation

### ISSUE THREAD

**Issue by zijinoier** [NVlabs/traffic-behavior-simulation#1]
Hi, when I follow the readme to Step 4 and Step 5: Closed-loop simulation.
It says `python scripts/evaluate.py \...`. However, the `evaluate.py` is not uploaded in `scripts` folder. 

P.S. Due to the large size of the nuscene or Lyft dataset, it is not easy to get started. Is it possible to provide a trained baseline model so that closed-loop simulation can be performed more easily?

**Comment by chenyx09** [comment]
Thank you for pointing it out, we accidentally added evaluate.py to .gitignore. It is fixed now. As for checkpoints, we added a link to pretrained nuScenes checkpoints in Readme.

**Comment by zijinoier** [comment]
After today's commit, I was able to  run the closed-loop simulation and produce videos like the one below. Thank you for your efforts 😀. 
![Screenshot from 2023-04-11 11-17-05](https://user-images.githubusercontent.com/18390668/231047651-0c763ac7-3254-4682-9661-60c356da694b.png)
Also, the video at the beginning of the `README`is different and has a 3D simulation interface. May I ask how to produce it? Is co-simulation with Omniverse or Carla required?

P.S. I noticed that the code requires placing the NuScenes Map expansion pack in the map folder. I think it would be nice to emphasize this in the data tree structure in the `README`, as follows:
```
nuscenes/
├── maps/
│   └── expansion/
│   v1.0-mini/
│   v1.0-trainval/
```
